keras_hub/src/tests/mocks/mock_gemma3_tokenizer.py

```diff
@@ -0,0 +1,155 @@
+import tensorflow as tf
+
+from keras_hub.src.tokenizers.tokenizer import Tokenizer
+from keras_hub.src.utils.tensor_utils import convert_to_ragged_batch
+from keras_hub.src.utils.tensor_utils import is_int_dtype
+from keras_hub.src.utils.tensor_utils import is_string_dtype
+from keras_hub.src.utils.tensor_utils import preprocessing_function
+
+
+class MockGemma3Tokenizer(Tokenizer):
```

Comment:
This mock is going to be hard to maintain. What we if just add the real gemma3 spm file to `tests/test_data/gemma3_vocab.spm`? And leave a todo to replace with a smaller test tokenizer?

I'd rather have a slightly oversized file than have to carry this code.

Reply:
I can move back to the original SPM file, but it's ~4 MB